docs/schema/records_reference.md

```diff
@@ -36,12 +36,22 @@ The following example demonstrates the package metadata and record fields.
 
 ## Record structure
 
-A record **must** contain an [ocid](identifiers.md#open-contracting-process-identifier-ocid) and all [releases](#releases) about the contracting process at the time of the record's publication. As such, a record functions as an index of releases about a contracting process.
+A record **must** contain an [ocid](identifiers.md#open-contracting-process-identifier-ocid) and it **should** contain either:
 
-A record **should** contain a [compiledRelease](#compiled-release) object, which represents the state of the contracting process at the time of the record's publication.
+* All [releases](#releases) about the contracting process at the time of the record's publication, and/or;
+
+* A [compiledRelease](#compiled-release) object, which represents the state of the contracting process at the time of the record's publication.
+
+As such, a record functions as an index of releases and/or as the latest version of a contracting process at the time of the record's publication.
 
 A record **may** contain a [versionedRelease](#versioned-release) object, which aggregates, into a single object, all values of all fields from all releases up to the time of the record's publication. The versioned release is designed to make it easy to see how values change from one release to another, and will often be generated by data users, rather than by publishers.
 
+```{admonition} Releases
```

Comment:
@jpmckinney I'm not sure if we should use `{versionadded} 1.2` here or just `{admonition}` (or nothing?), as in this case we are not adding any new field, but removing an (important!) constraint.